ProjectShop/ProjectShop/urls.py

```diff
@@ -14,8 +14,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
+from django.urls import include
 from django.urls import path
```

Comment:
I guess, it would be better to do it this way:
from django.urls import path, include